installing/installing_openstack/deploying-openstack-with-rootVolume-etcd-on-local-disk.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="deploying-openstack-on-local-disk"]
+= Deploying OpenStack with rootVolume and etcd on local disk
+include::_attributes/common-attributes.adoc[]
+:context: deploying-openstack-on-local-disk
+
+toc::[]
+
+include::snippets/technology-preview.adoc[]
+
+As a day 2 operation, by moving etcd from a root volume (Cinder) to a dedicated ephemeral local disk, you can resolve and prevent performance issues of your {rh-openstack-first} installation.
+
+include::modules/installation-osp-local-disk-deployment.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
I want this to appear following this module and not at the end of the assembly so I'm using the `.Additional resources` rather than the === style here.